### PR TITLE
adding debugging/troubleshooting logs

### PR DESCRIPTION
**Pull request by anandgupta42**
## Overview
- Adding debugging/troubleshooting logs for the extension

## Checklist

- [X ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

**Comment by mdesmet** [comment]
FYI Now with the external executor, the CustomUnknownException  might not be anymore valid as the main/potentially failing logic is not always executed in Python. Before it was usually a right assumption if there is an exception that is not a PythonException it is a bug in the code and not just an environment issue. Please take that into account. Maybe we need tor rename that class to CustomJavascriptException so that there is no confusion about it.

File: src/dbt_client/index.ts [hunk 1]
```diff
@@ -112,12 +113,18 @@ export class DBTClient implements Disposable {
...
+          message = "Please ensure dbt cloud cli is installed.";
+          window.showErrorMessage("Please ensure dbt cloud cli is installed.");
+        } else {
+          const answer = await window.showErrorMessage(
+            "Please ensure dbt is installed.",
+            DbtInstallationPromptAnswer.INSTALL,
+          );
+          if (answer === DbtInstallationPromptAnswer.INSTALL) {
+            commands.executeCommand("dbtPowerUser.installDbt");
+          }
```
Comment (mdesmet) [inline src/dbt_client/index.ts:127]: I think we can comment this for now as dbt cannot yet be installed. Add a TODO

File: src/manifest/dbtProject.ts [hunk 2]
```diff
@@ -625,7 +629,7 @@ select * from renamed
       window.showErrorMessage("Please enter a positive number for query limit");
       return;
     }
-    this.telemetry.sendTelemetryEvent("executeSQL", {
+    this.terminal.info("executeSQL", query, true, {
```
Comment (mdesmet) [inline src/manifest/dbtProject.ts:632]: Change to debug

File: src/webview_provider/altimateWebviewProvider.ts [hunk 3]
```diff
@@ -200,7 +204,11 @@ export class AltimateWebviewProvider implements WebviewViewProvider {
           break;
       }
     } catch (err) {
-      this.dbtTerminal.debug("error while handling command", err);
+      this.dbtTerminal.debug(
```
Comment (mdesmet) [inline src/webview_provider/altimateWebviewProvider.ts:207]: Shouldn't this be error?

File: src/dbt_client/index.ts (hunk 1 above)
Comment (AdiGajbhiye) [reply src/dbt_client/index.ts:127]: currently in `master`, it shows `Install dbt` button in both cases. This shows the button only for core

**Review by mdesmet — APPROVED** [review]

**Pull request merged**